modules/gdscript/gdscript_parser.cpp

```diff
@@ -461,7 +461,7 @@ GDScriptParser::Node *GDScriptParser::_parse_expression(Node *p_parent, bool p_s
 				if (for_completion && ScriptCodeCompletionCache::get_sigleton()) {
 					res = ScriptCodeCompletionCache::get_sigleton()->get_cached_resource(path);
 				} else {
-					res = ResourceLoader::load(path);
+					res = ResourceLoader::load(path, "", false, true); // suppress errors
```

Comment:
Instead of doing all the above hack, why not simply doing
```
if (FileAccess:exists(path)) {
   res = ResourceLoader::load(path);
}
```

Reply:
Quite true, changed.